landing/styles/docs.css

```
@@ -0,0 +1,160 @@
+  --ink: #07111f;
+  --ink-2: #263241;
+  --ink-3: #5d6877;
+  --ink-4: #8a94a3;
+  --blue: #0969da;
+  --green: #008a55;
+  --line: #d9dee8;
+  --line-soft: #edf0f5;
+  --paper: #ffffff;
+  --paper-2: #fbfcfe;
+  --paper-3: #f5f7fb;
+  --font-sans: "Inter", "Noto Sans JP", -apple-system, BlinkMacSystemFont, system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)